### ISSUE TITLE
Changes to offal broke sweetbread recipe

### ISSUE THREAD

**Issue by Ker-Nes** [CleverRaven/Cataclysm-DDA#26876]
**Describe the bug**  
The recent changes to offal broke the sweetbread recipe. Now, instead of the old recipe, it produce "Raw Sweetbread", which has considerably more nutrition but rots far sooner.

**To Reproduce**  
1. Know Sweetbread recipe.
2. Have Sweetbread ingredients.
3. Make Sweetbread recipe.

**Expected behavior**  
It making the old cooked sweetbread or the recipe being removed in preparation for the new changes related to offal.

**Versions:**
0.C-34855-gdb7bcb4 (tiles)

**Comment by KorGgenT** [comment]
fixed in #26873

**Comment by AMurkin** [comment]
Now we have at map load in debug.log:
```
	debug_write_backtrace(std::ostream&)
	.../src/debug.cpp:577
	DebugLog(DebugLevel, DebugClass)
	.../src/debug.cpp:732
	realDebugmsg(char const*, char const*, char const*, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)
	.../src/debug.cpp:85
	string_id<recipe>::obj() const
	/usr/include/c++/7/bits/basic_string.h:647
	void realDebugmsg<char const*>(char const*, char const*, char const*, char const*, char const*&&)
	.../src/debug.h:76
	string_id<recipe>::obj() const
	.../src/recipe_dictionary.cpp:51
	deserialize(recipe_subset&, JsonIn&)
	.../src/savegame_json.cpp:2751
	player::deserialize(JsonIn&)
	.../src/json.h:283
	??
	.../src/json.h:742
	player::deserialize(JsonIn&)
	.../src/savegame_json.cpp:794
	game::unserialize(std::istream&)
	.../src/json.h:294
	??
	.../src/json.h:742
	game::unserialize(std::istream&)
	.../src/savegame.cpp:254
	read_from_file(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::function<void (std::istream&)> const&)
	.../src/cata_utility.cpp:430
	game::load(save_t const&)
	/usr/include/c++/7/bits/basic_string.h:647 (discriminator 6)
	game::load(save_t const&)
	.../src/game.cpp:2633 (discriminator 6)
	main_menu::load_character_tab()
	.../src/main_menu.cpp:952
	main_menu::opening_screen()
	.../src/main_menu.cpp:476
	main
	.../src/main.cpp:669
	__libc_start_main
	/build/glibc-OTsEL5/glibc-2.27/csu/../csu/libc-start.c:344
	_start
	??:?
src/recipe_dictionary.cpp:51 [const T& string_id<T>::obj() const [with T = recipe]] invalid recipe id "sweetbread"
```
